packages/collaboration-extension/src/filebrowser.ts

```diff
@@ -160,6 +161,42 @@ export const defaultFileBrowser: JupyterFrontEndPlugin<IDefaultFileBrowser> = {
   }
 };
 
+/**
+ * The default collaborative drive provider.
+ */
+export const logger: JupyterFrontEndPlugin<void> = {
+  id: '@jupyter/collaboration-extension:logger',
+  description: 'A logging plugin for debugging purposes.',
+  autoStart: true,
+  requires: [ITranslator],
+  optional: [],
+  activate: (app: JupyterFrontEnd, translator: ITranslator): void => {
+    const trans = translator.load('jupyter_collaboration');
+
+    app.serviceManager.events.stream.connect((_, emission) => {
+      if (
+        emission.schema_id ===
+        'https://events.jupyter.org/jupyter_server/jupyter_collaboration/v1'
+      ) {
+        console.debug(
```

Comment:
Would you mind switching to the logger that is a perfect fit for this case: see https://github.com/jupyterlab/extension-examples/tree/4.0/log-messages